vespajlib/src/main/java/com/yahoo/concurrent/SystemTimer.java

```diff
@@ -32,8 +32,12 @@ public static int detectHz() {
         return hz;
     }
 
+    public static int adjustTimeoutByDetectedHz(int timeoutMS) {
```

Comment:
Should we explicitly mention the time unit in the name to make it more obvious what unit is used? E.g. `adjustMsTimeoutByDetectedHz` or `adjustTimeoutMsByDetectedHz`?

Reply:
I used a typesafe Duration instead.